firebase-perf/e2e-app/src/androidTest/java/com/google/firebase/testing/fireperf/FirebasePerformanceScreenTracesTest.java

```diff
@@ -41,6 +48,16 @@
           /* initialTouchMode= */ false,
           /* launchActivity= */ true);
 
+  @After
+  public void pressHome_toTriggerSendScreenTrace() {
```

Comment:
Are we triggering an event sent for every app backgrounding? I thought we were dispatching events scheduled every 30 seconds managed by Firelog service. @jeremyjiang-dev